api/v1alpha/location_types.go

```diff
@@ -70,6 +70,11 @@ type LocationStatus struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Class",type="string",JSONPath=".spec.locationClassName"
+// +kubebuilder:printcolumn:name="City",type="string",JSONPath=".spec.topology['topology.datum.net/city-code']"
```

Comment:
This format doesn't seem to work for me locally - is it working for you?

The following does work:

```suggestion
// +kubebuilder:printcolumn:name="City",type="string",JSONPath=`.spec.topology.topology\.datum\.net/city-code`
```

Reply:
You are right. My mistake was not actually checking the value. I thought that if it didn’t fail to build or install the CRD, the column would just work. 

Before:
```
kubectl get locations
NAME            CLASS           CITY   AGE
us-central1-a   datum-managed          24d
```

After: 

```
NAME            CLASS           CITY          AGE
us-central1-a   datum-managed   us-central1   24d
```